platforms/yttrex/backend/bin/leaves-parser.ts

```diff
@@ -72,6 +73,10 @@ const run = async (): Promise<void> => {
       write: mongoW,
     };
 
+    const errorReporter = FixtureReporter(
+      path.resolve(process.cwd(), '__tests__/fixtures/htmls')
```

Comment:
Can we make the `"parsers/__tests__/fixtures"` string a constant variable or something?

Reply:
Done